src/Symfony/Component/Debug/Tests/Fixtures/FinalClass.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Symfony\Component\Debug\Tests\Fixtures;
+
+/**
+ * @final since version 3.3.
+ */
+class FinalClass
```

Comment:
but we don't care: if the class is really final, then there is no way we end up in the code path where that final class is the parent of something else, isn't it?